subjects/fifthandskip/README.md

```diff
@@ -0,0 +1,44 @@
+## fifthandskip
+
+### Instructions
+
+Write a function `FifthAndSkip()` that takes a `string` and returns another `string`. The function separates every five characters of the `string` with a space and removes the sixth one.
+
+- If there is a space in the middle of a word it should ignore it and get the first character until getting to a length of 5.
+- If the `string` is less than 5 characters returns "Invalid Output\n".
```

Comment:
fix: Add condition for empty string, If the `string` is empty or less than 5 ...

Reply:
Thank you, it is fixed.